src/main/webapp/IFrame.js

```diff
@@ -12,143 +12,163 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-let videoUpdating; // boolean for when server request is updating video
-let player;
+let videoUpdating; // is video currently updating to match Firestore info?
+const SYNC_WINDOW = 5; // max time diff between client and Firestore
+
+const firebaseConfig = {
+  apiKey: API_KEY, // eslint-disable-line no-undef
+  authDomain: 'lounge-95f01.firebaseapp.com',
+  databaseURL: 'https://lounge-95f01.firebaseio.com',
+  projectId: 'youtube-lounge',
+  storageBucket: 'youtube-lounge.appspot.com',
+  messagingSenderId: '681171972170',
+  appId: '1:681171972170:web:4c6526b8eb788af9d876b3',
```

Comment:
Is any of this unique to the app and be considered private information?

Reply:
Yeah, I'll move it to a private file.